views/teams/edit-team.hbs

```diff
@@ -35,7 +35,7 @@
                         <div class="form-group">
                             <label>
                                 <input type="checkbox" name="messenger" value="true" {{#inArray "messenger" ../team.features}}checked{{/inArray}}>
-                                Messenger für Team aktivieren
+                                Teilnahmer:innen erlauben im Team Chat Nachrichten zu verfassen
```

Comment:
use similar language than for videoconferences add :innen in other text parts or remove it here